src/dashboards/videos/details.jsonnet

```
@@ -119,3 +136,131 @@ dashboard.new(
   ),
   gridPos={ x: 0, y: 0, w: 12, h: 9 }
 )
+.addPanel(
+  statPanel.new(
+    title='Views count by time threshold',
```

I think we have to be more synthetic in titles as we have a complete description!

```suggestion
    title='Views',
```

---

Ok) Thanks)